Task05/Task05/Program.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task05
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            double x = Sinuss(2,3) + Sinuss(5,7) + Sinuss(11,13);
+            Console.WriteLine("x = " + x);
+            Console.ReadKey();
+        }
+        static double Sinuss(double x,double y)
+        {
+            return Math.Round(((Math.Sin(x)+Math.Sin(y))/(x+y)),3);
```

Comment:
Округлять не нужно. Метод должен вычислять значение максимально точно. Иначе в большим выражении, когда метод используется несколько раз, ошибка округления будет накапливаться.
```suggestion
            return (Math.Sin(x)+Math.Sin(y))/(x+y);
```